.travis.yml

```diff
@@ -21,9 +21,9 @@ matrix:
         # - env: TOXENV=py35
         #   python: 3.5
         - env: TOXENV=py36
-          python: 3.6.10 # We're targeting a specific patch version; if set to 3.6, the GitHub Travis build will use Python 3.6.7 and will fail due to a cryptography installation error
+          python: 3.6.15 # We're targeting a specific patch version; if set to 3.6, the GitHub Travis build will use Python 3.6.7 and will fail due to a cryptography installation error
         - env: TOXENV=py37
-          python: 3.7.11
+          python: 3.7.13
```

Comment:
Increasing the Python 3.7 version used in Travis from `3.7.11` fixes an issue where `pipenv` ignores `PIPENV_YES=True`, causing the build to hang.

Also, increasing the Python 3.6 and 3.7 versions resolve Trivy vulnerabilities.